README.md

```diff
@@ -53,22 +56,12 @@ git clone https://github.com/sassoftware/viya4-iac-azure
 cd viya4-iac-azure
 ```
 
-### Authenticating Terraform to access Azure
-
-See [Terraform Azure Authentication](./docs/user/TerraformAzureAuthentication.md) for details.
-
-### Customize Input Values
-
-Create a file named `terraform.tfvars` to customize any input variable value. For starters, you can copy one of the provided example variable definition files in `./examples` folder. For more details on the variables declared in [variables.tf](variables.tf) refer to [CONFIG-VARS.md](docs/CONFIG-VARS.md).
-
-When using a variable definition file other than `terraform.tfvars`, see [Advanced Terraform Usage](docs/user/AdvancedTerraformUsage.md) for additional command options.
-
-## Creating and Managaging the Cloud Resources
+## Run -- Creating and Managaging the Cloud Resources
```

Comment:
The name does not fit. The old name "Creating and Managing the Cloud Resources" is preferred. Also, why were the the "Authenticating Terraform to access Azure" and "Customize Input Values" sections removed. That info is needed. If you could re-add them

Reply:
For the variable definition bits, that information is found under the "running ..." steps, so it seemed repetitive.  I have added it back in if you prefer it in both places.  

From a user just running it, it was confusing that the TF variable information is in two places and I prefer to just get to the running section and that running section contains the TF variables ...

I have reverted the name, but clarified in the actions that the user is doing the action by running.